Documentation/upgrade.md

```diff
@@ -4,47 +4,68 @@ weight: 60
+those releases.
+- [Upgrade 0.7 to 0.8](https://rook.io/docs/rook/v0.8/upgrade.html)
 - [Upgrade 0.6 to 0.7](https://rook.io/docs/rook/v0.7/upgrade.html)
 - [Upgrade 0.5 to 0.6](https://rook.io/docs/rook/v0.6/upgrade.html)
 
 ### Patch Release Upgrades
-The changes in patch releases are scoped to the minimal changes necessary and are expected to be straight forward to upgrade.
-For upgrades of 0.8 patch releases such as `0.8.0` to `0.8.1`, see the [patch release upgrade guide](upgrade-patch.md).
+One of the goals of the 0.9 release is that patch releases are able to be automated completely by
+the Rook operator. It is intended that upgrades from one patch release to another are as simple as
+updating the image of the Rook operator. For example:
+```
+kubectl -n rook-ceph-system set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v0.9.x
+```
 
 ## Considerations
-With this manual upgrade guide, there are a few notes to consider:
-* **WARNING:** Upgrading a Rook cluster is a manual process in its very early stages.  There may be unexpected issues or obstacles that damage the integrity and health of your storage cluster, including data loss.  Only proceed with this guide if you are comfortable with that.
-* This guide assumes that your Rook operator and its agents are running in the `rook-system` namespace. It also assumes that your Rook cluster is in the `rook` namespace.  If any of these components is in a different namespace, search/replace all instances of `-n rook-system` and `-n rook` in this guide with `-n <your namespace>`.
-  * New Ceph specific namespaces (`rook-ceph-system` and `rook-ceph`) are now used by default in the new release, but this guide maintains the usage of `rook-system` and `rook` for backwards compatibility.  Note that all user guides and examples have been updated to the new namespaces, so you will need to tweak them to maintain compatibility with the legacy `rook-system` and `rook` namespaces.
+With this upgrade guide, there are a few notes to consider:
+* **WARNING:** Upgrading a Rook cluster is not without risk. There may be unexpected issues or
+  obstacles that damage the integrity and health of your storage cluster, including data loss. Only
+  proceed with this guide if you are comfortable with that.
+* Rook is able to handle a great deal of the upgrade on its own, but manual steps are still
+  necessary. This is in part to reduce the Kubernetes privileges required by the Rook operator.
+* This guide assumes that your Rook operator and its agents are running in the `rook-system`
+  namespace. It also assumes that your Rook cluster is in the `rook` namespace. If any of these
+  components is in a different namespace, search/replace all instances of `-n rook-system` and
+  `-n rook` in this guide with `-n <your namespace>`.
+  * New Ceph specific namespaces (`rook-ceph-system` and `rook-ceph`) are now used by default in the
```

Comment:
you could remove this bullet, it was specific to v0.8

Reply:
And because of the above bullet, the guide still stays as `rook` and `rook-system`, yes?

Reply:
actually, the previous bullet would be good to keep except that our default namespaces are as in this bullet that I suggested could be deleted. So the previous bullet should mention `rook-ceph` and `rook-ceph-system` are the default namespaces.